src/Listener/ToolbarListener.php

```diff
@@ -53,19 +53,15 @@ class ToolbarListener implements ListenerAggregateInterface
      */
     public const DOC_URI = 'https://docs.laminas.dev/';
 
-    /** @var object */
-    protected $renderer;
-
     /** @var Options */
     protected $options;
 
     /**
-     * @param object  $viewRenderer
+     * @param object $renderer
      */
-    public function __construct($viewRenderer, Options $options)
+    public function __construct(protected $renderer, Options $options)
```

Comment:
Yes, `$renderer` can be promoted property, since the parameter type is mixed (not typed)
and `Options` need to be kept as is as property not typed, and parameter typed.

Ref https://3v4l.org/Ird4g vs https://3v4l.org/bHo6B